src/common/settingsMapper.ts

```diff
@@ -1,8 +1,16 @@
 import Handsontable from 'hot-alias';
+import { Props } from './index';
 
 export class SettingsMapper {
+
+  // TODO: this should be removed after the release of Handsontable 5.0.0+. It's a workaround for a faulty Handsontable definition file.
+  private hooks: any;
+
```

Comment:
This empty line isn't necessary.

Reply:
I'll keep it to separate the workaround from the "actual code", it'll be easier to notice after releasing HOT 5.0.0+.

Reply:
:+1: